src/components/organisms/AuthenticationModal/InitialForm/InitialForm.tsx

```diff
@@ -21,12 +21,12 @@ export const InitialForm: FC<InitialFormProps> = ({
       {/*  Create your account*/}
       {/*</div>*/}
       <div className="buttons-separator">or</div>
-      <div
-        className="btn btn-block btn-centered login-button"
-        onClick={displayLoginForm}
-      >
-        Log in for non-Okta users
-      </div>
+      <p className="login-text">
+        No Okta?{" "}
+        <span className="login-button" onClick={displayLoginForm}>
+          Login here
+        </span>
```

Comment:
Pretty sure we have explicit styling for `button` that allows displaying as a text link for this purpose. Using `span` here isn't really valid/semantic/accessible.